docs/docs/cmd/aad/app/app-open.md

```diff
@@ -0,0 +1,42 @@
+# aad app open
+
+Returns deep link to open the Azure portal on the Azure AD app registration management page.
+
+## Usage
+
+```sh
+m365 aad app open [options]
+```
+
+## Options
+
+`--appId <appId>`
+: Application (client) ID of the Azure AD application registration to open.
+
+`--preview`
+: Use to open the url of the Azure AD preview portal.
+
+`--autoOpenBrowser`
```

Comment:
Based on what we discussed in https://github.com/pnp/cli-microsoft365/issues/3118, shall we remove this option in favor of the global config setting?

Reply:
Jep, I think that would be best. I will update the code and push it for review again.

Reply:
Awesome! Thank you 👍

Reply:
Oh wait, the setting is still called `autoOpenBrowserOnLogin`. I think I should wait until it is renamed. Is that already in the pipeline?

Reply:
I think that task is still open 😬

Reply:
In that case let's just keep this pull request open until it is. I'm in no hurry 😊

Reply:
...or let me fix it myself 😂

Reply:
> Based on what we discussed in #3118, shall we remove this option in favor of the global config setting?

Haven't we agreed to remove this option?